bin/paragon-scripts.js

```diff
@@ -1,26 +1,68 @@
 #!/usr/bin/env node
+/* eslint-disable no-console */
```

Comment:
You turned off the linter for the entire page, even though you're only using `console.log` in one place. Should I turn off the linter rule only in one place?

Reply:
it is used twice there but I agree for two cases it is good idea.